app/views/step_by_step_pages/show/_required_actions.html.erb

```diff
@@ -1,3 +1,9 @@
+<% if @step_by_step_page.status.submitted_for_2i? %>
+  <%= render "govuk_publishing_components/components/inset_text" do %>
+    <p class="govuk-body">Not yet claimed for 2i</p>
+  <% end %>
+<% end %>
```

Comment:
This will lead to having two inset prompts in the page, if for example it is submitted for 2i and then a step is tweaked. I can see why you've kept it separate, as we don't really want to rewrite the `should_show_required_prepublish_actions` wrapper on line 7.

Not sure if that's an issue from a user perspective - but it probably is a problem from the tests' perspective, which are only expecting one inset_text component at the moment and will lead to an "ambiguous match" if both are rendered. Other components let you pass an `id` to remove the ambiguity (see the `summary-list` usage in `show.html`) but I don't think we can do that out of the box here 🤔

Reply:
As discussed, we'll keep two separated inset text (as per design and because of a lack in new copy to work with) but re-placed the 2i one after the required actions to make sure tests are failing.